doc/reference/trd104-syscalls.md

```diff
@@ -522,6 +522,12 @@ on RISC-V.
 | Address          | r2       |
 | Size             | r3       |
 
+The *buffer number* argument is an ordinal number (index) of the buffer.
```

Comment:
Is there some other place in which the terminology "buffer number" is used in Tock? If you're proposing to introduce it, I'm afraid it might be even more confusing than what we have currently. "buffer" is a rather overloaded term. Traditionally we've been calling these specifiers "subdriver number" or, for each system call respectively, "allow number" and "subscribe number". This makes it obvious that you're referring to the identifier of a given allow number provisioned by a driver as part of its ABI.

Reply:
I just copied this from the argument name for consistence. It's only used in doc/reference/trd-userspace-readable-allow-syscalls.md apart from here.

Changed to "allow number".